docs/md_templates/_common_body.tpl

```diff
@@ -168,7 +168,9 @@ We're excited to see so many articles popping up on data ethics! The short list
 
 ## Where things have gone wrong
 
-To make the ideas contained in the checklist more concrete, we've compiled [examples](http://deon.drivendata.org/examples/) of times when things have gone wrong. They're paired these with the checklist questions to help illuminate where in process ethics discussions may have helped provide a course correction.
+To make the ideas contained in the checklist more concrete, we've compiled [examples](http://deon.drivendata.org/examples/) of times when things have gone wrong. They're paired these with the checklist questions to help illuminate where in the process ethics discussions may have helped provide a course correction.
```

Comment:
> They're paired these

Still has typo?

Reply:
lol. well this PR will have *two* bonus typo fixes

Reply:
💸